src/main/java/net/neoforged/neoforge/fluids/FluidUnits.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.fluids;
+
+import net.minecraft.resources.ResourceLocation;
+
+public class FluidUnits {
+    public static final FluidUnit BUCKET = c("bucket");
+    public static final FluidUnit BOTTLE = c("bottle");
+    public static final FluidUnit BOWL = c("bowl");
+
+    public static final FluidUnit BLOCK = c("block");
+    public static final FluidUnit INGOT = c("ingot");
+    public static final FluidUnit NUGGET = c("nugget");
+    public static final FluidUnit GEM = c("gem");
+
+    public static final FluidUnit SLIMEBALL = c("slimeball");
+    public static final FluidUnit BRICK = c("brick");
+    public static final FluidUnit PANE = c("pane");
+    public static final FluidUnit MB = c("mb");
```

Comment:
Nitpick, but `MB` should really be at the top of this file with `BUCKET`, those are the two most essential units.